nimble/host/mesh/src/proxy.c

```diff
@@ -808,7 +808,8 @@ void bt_mesh_proxy_gatt_disconnect(void)
 	for (i = 0; i < ARRAY_SIZE(clients); i++) {
 		struct bt_mesh_proxy_client *client = &clients[i];
 
-		if ((client->conn_handle != BLE_HS_CONN_HANDLE_NONE) && (client->filter_type == WHITELIST || 
+		if ((client->conn_handle != BLE_HS_CONN_HANDLE_NONE) &&
```

Comment:
Please align each line to the beginning of the condition in the previous line, like this:

```
		if ((client->conn_handle != BLE_HS_CONN_HANDLE_NONE) &&
		    (client->filter_type == WHITELIST ||
		     client->filter_type == BLACKLIST)) {
			client->filter_type = NONE;
```
And then squash the style fix commits.
Thanks!